projectforum/projects/templates/project_detail.html

```diff
@@ -9,6 +9,7 @@
     <script src="{% static 'external/jqueryraty/jquery.raty.js' %}"></script>
     <script src="{% static 'js/project-detail.js' %}"></script>
     <script src="{% static 'js/ratings.js' %}"></script>
+    <script src="//cdn.jsdelivr.net/jquery.validation/1.14.0/jquery.validate.min.js"></script>
```

Comment:
Can we download this and put it in the external folder? That way whitenoise can deal with it appropriately.